app/components/AccessNameCell.tsx

```diff
@@ -7,28 +7,9 @@
  */
 import type { CellContext } from '@tanstack/react-table'
 
-import type { IdentityType } from '@oxide/api'
-
-import { Badge } from '~/ui/lib/Badge'
-
 /**
  * Display the user or group name. If the row is for a group, add a GROUP badge.
  */
-export const AccessNameCell = <
-  RowData extends { name: string; identityType: IdentityType },
->(
+export const AccessNameCell = <RowData extends { name: string }>(
   info: CellContext<RowData, string>
-) => {
-  const name = info.getValue()
-  const identityType = info.row.original.identityType
-  return (
-    <>
-      <span>{name}</span>
-      {identityType === 'silo_group' ? (
-        <Badge color="neutral" className="ml-2">
-          Group
-        </Badge>
-      ) : null}
-    </>
-  )
-}
+) => <span>{info.getValue()}</span>
```

Comment:
Most likely this doesn't need to exist, unless you need the `<span>` there for some reason. If you leave `cell` unspecified, React Table will just stick `info.getValue()` in there for you.